mipengine/node/monetdb_interface/common_actions.py

```diff
@@ -190,14 +189,7 @@ def get_table_data(table_name: str) -> List[ColumnData]:
     """
     schema = get_table_schema(table_name)
 
-    row_stored_data = db_execute_and_fetchall(
-        f"""
-        SELECT {table_name}.*
-        FROM {table_name}
-        INNER JOIN tables ON tables.name = '{table_name}'
-        WHERE tables.system=false
-        """
-    )
+    row_stored_data = db_execute_and_fetchall(f"""SELECT * FROM {table_name}""")
```

Comment:
This change should be reverted.

It's not secure to do it like that until this https://team-1617704806227.atlassian.net/browse/MIP-133